extension/scripts/global/functions/torn.js

```diff
@@ -1650,6 +1650,14 @@ function isOwnProfile() {
 	return (params.has("XID") && parseInt(params.get("XID")) === id) || (params.has("NID") && params.get("NID") === name);
 }
 
+function getUserE() {

```

Comment:
Same here as with the feature name before, don't shorten it.

Reply:
It's fine for variables inside a feature, but global functions should be as descriptive as possible.